src/containers/AccountConnection.jsx

```diff
@@ -37,10 +37,10 @@ class AccountConnection extends Component {
   connectAccount (auth) {
     const { folderPath } = auth
     const { connector } = this.props
-    let { account } = this.state
+    let { account, successTimeout } = this.state
```

Comment:
0 is tricky from my point of view, as it will be used for a `setTimeout` that we might want to use with 0ms.